test/event_emitter.js

```diff
@@ -0,0 +1,22 @@
+import { EventEmitter } from '../src/util/event_emitter';
+
+describe('EventEmitter', function() {
+  beforeEach(function() {
+    this.emitter = new EventEmitter();
+  });
+
+  describe('when a listener is added', function() {
```

Comment:
and test for removing listener?

Reply:
Yes tests are still missing. I couldn't  find the time to write them yet 😓 That's why I added the **wip** label to the PR